graphics/gscan2pdf/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           perl5 1.0
+
+name                gscan2pdf
+version             2.12.4
+revision            0
+
+categories          graphics
+maintainers         nomaintainer
+license             GPL-3
+description         A GUI to produce PDFs or DjVus from scanned documents
+long_description    ${description}
+
+homepage            http://gscan2pdf.sourceforge.net
+master_sites        sourceforge
+use_xz              yes
+
+checksums           rmd160  d800f0f54214763dacf0847f5eece92dcd076b65 \
+                    sha256  52b06dd1091293b20fe26658168c4534943542671be7709e9af9587ecc6367fb \
+                    size    503792
+
+depends_lib         port:ImageMagick \
+                    port:librsvg \
+                    port:p${perl5.major}-config-general \
+                    port:p${perl5.major}-date-calc \
+                    port:p${perl5.major}-filesys-df \
+                    port:p${perl5.major}-goocanvas2 \
+                    port:p${perl5.major}-gtk3 \
+                    port:p${perl5.major}-gtk3-imageview \
+                    port:p${perl5.major}-gtk3-simplelist \
+                    port:p${perl5.major}-image-sane \
+                    port:p${perl5.major}-ipc-system-simple \
+                    port:p${perl5.major}-list-moreutils \
+                    port:p${perl5.major}-locale-gettext \
+                    port:p${perl5.major}-log-log4perl \
+                    port:p${perl5.major}-pdf-builder \
+                    port:p${perl5.major}-perlmagick \
+                    port:p${perl5.major}-proc-processtable \
+                    port:p${perl5.major}-readonly \
+                    port:p${perl5.major}-set-intspan \
+                    port:p${perl5.major}-sub-override \
+                    port:p${perl5.major}-try-tiny \
+                    port:sane-backends \
+                    port:tiff \
+
+configure.cmd       ${perl5.bin}
+configure.pre_args  Makefile.PL --prefix=${prefix}
+
+post-destroot {
+    ln -s ${prefix}/libexec/perl${perl5.major}/sitebin/gscan2pdf \
+        ${destroot}${prefix}/bin/
+    eval delete [glob ${destroot}${prefix}/lib/perl5/*/*/perllocal.pod]
```

Comment:
there is no reason to do a `glob` here since there is only one file that you want to remove; also as `port lint` points out you shouldn't use `eval`. So just remove the one file, presumably with (but please double-check as I haven't tried it):
``` file delete ${destroot}${prefix}/lib/perl5/${perl5.major}/darwin-thread-multi-2level/perllocal.pod```

and please add a comment of why this was needed (i.e., conflict with the a file that has the same name but belongs to the `ossp-uuid` port) and perhaps a link to the Trac tickets you mentioned.

Reply:
Well I copied the code from other ports that do it that way. I can't check in how many ports this is used right now but I got the feeling that this is standard in some perl ports. An advice of perl port maintainers would be welcome on what to do.

Reply:
I  just `grep`-ed for `poerllocal.pod` in the MacPorts tree and there are no ports that use `eval` as far as I can tell, so you'll need to correct that for sure. And since here there is only one file that you need to remove I would just specify it explicitly

Reply:
Ok. Thanks for looking.
I created ticket https://trac.macports.org/ticket/64444 in the meantime.

Reply:
Updated as per your recommendation (it installs fine)